src/test/java/salesforce/CreateAnAssetTest.java

```diff
@@ -0,0 +1,120 @@
+    driver.findElement(By.xpath("(//div[contains(@class,\'uiInputDate\')]/div/input)[1]")).sendKeys(vars.get("installDate").toString());
+    driver.findElement(By.xpath("(//div[contains(@class,\'uiInputDate\')]/div/input)[2]")).sendKeys(vars.get("purchaseDate").toString());
+    driver.findElement(By.xpath("(//div[contains(@class,\'uiInputDate\')]/div/input)[3]")).sendKeys(vars.get("usageDate").toString());
+    driver.findElement(By.xpath("(//div[@class=\'autocompleteWrapper slds-grow\']/input)[1]")).click();
+    wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("(//div[@role=\'listbox\']//a)[1]")));
+    driver.findElement(By.xpath("(//div[@role=\'listbox\']//a)[1]")).click();
+    driver.findElement(By.xpath("(//div[@class=\'autocompleteWrapper slds-grow\']/input)[2]")).click();
+    wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("(//div[@class=\'listContent\']//a)[3]")));
+    driver.findElement(By.xpath("(//div[@class=\'listContent\']//a)[3]")).click();
+    driver.findElement(By.xpath("(//div[@class=\'autocompleteWrapper slds-grow\']/input)[3]")).click();
+    wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("(//div[@class=\'listContent\']//a)[7]")));
+    driver.findElement(By.xpath("(//div[@class=\'listContent\']//a)[7]")).click();
+    driver.findElement(By.xpath("//button[@title=\'Guardar\']")).click();
+    wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//h1/div/span")));
+    Assertions.assertEquals (driver.findElement(By.xpath("//h1/div/span")).getText(), (vars.get("assetName").toString()));
+    delete();
+    logout();
+  }
+
+  @Test
+  public void necessaryAttributes13() {
+    login();
+    vars.put("asset", "Activos");
+    vars.put("assetName", "nombre 6");
+    driver.get("https://none773-dev-ed.lightning.force.com/lightning/o/Asset/list?filterName=Recent//");
+    wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".forceActionLink > div")));
+    driver.findElement(By.cssSelector(".forceActionLink > div")).click();
+    wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("(//div[contains(@class, \'uiInputText\')][.//label])[1]/input")));
+    driver.findElement(By.xpath("(//div[contains(@class, \'uiInputText\')][.//label])[1]/input")).sendKeys(vars.get("assetName").toString());
+    driver.findElement(By.xpath("(//div[@class=\'autocompleteWrapper slds-grow\']/input)[1]")).click();
+    wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("(//div[@role=\'listbox\']//a)[1]")));
+    driver.findElement(By.xpath("(//div[@role=\'listbox\']//a)[1]")).click();
+    driver.findElement(By.xpath("//button[@title=\'Guardar\']")).click();
+    wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//h1/div/span")));
+    Assertions.assertEquals (driver.findElement(By.xpath("//h1/div/span")).getText(), (vars.get("assetName").toString()));
+    delete();
+    logout();
+  }
+
+  public void logout() {
```

Comment:
I think that your methods' login, logout and delete should be change for test methods like @beforeClass, @AfterClass and @AfterEach

Reply:
You 're right. I did it